src/Plugin.php

```diff
@@ -51,12 +51,38 @@ public function handleArguments(array $arguments): array
             if (str_starts_with($argument, '--duration=')) {
                 $run->duration((int) str_replace('--duration=', '', $argument));
             }
-        }
 
-        foreach ($arguments as $argument) {
             if (str_starts_with($argument, '--concurrency=')) {
                 $run->concurrently((int) str_replace('--concurrency=', '', $argument));
             }
+
+            if (str_starts_with($argument, '--method=')) {
+                $run->method(str_replace('--method=', '', $argument));
+            }
+
+            if ($argument === '--get') {
+                $run->get();
+            }
+
+            if ($argument === '--post') {
```

Comment:
Can you receive the payload on the post's argument instead?

Reply:
Done: https://github.com/pestphp/pest-plugin-stressless/pull/8/commits/b7a9490b9e1aa33394f9a9ef3a396f344ddfdf9d